HW06.md

```diff
@@ -0,0 +1,1131 @@
+HW06
+================
+
+Steps:
+
+1.  Read in the 45 processed RNA-Seq data found in
+    “./RNA\_Seq\_processed”
+2.  Map gene.ids to gene.symbols (which I have for you)
+3.  For 27 genes of interest AND your favorite gene, perform a t-test to
+    see if it is differentially expressed between the WT vs H3K27M
+    samples
+4.  Create a graphing function and then create a boxplot that graphs
+    expression between the two groups
+
+#### Code
+
+``` r
+#create list of files contained by each directory 
+H3K27M_list <- list.files("/Users/JuliaShangguan/Desktop/R coding class/HW06/RNA_Seq_processed/H3K27M")
+WT_list <- list.files("/Users/JuliaShangguan/Desktop/R coding class/HW06/RNA_Seq_processed/WT")
+```
+
+``` r
```

Comment:
For readability, if you ever want the code chunk to not display these long messages like you see here, just add " message = FALSE " on this line and they won't show up